scripts/appstore-fixture-up.sh

```
@@ -0,0 +1,93 @@
+cd "${ROOT_DIR}"
+
+if [ "${APPSTORE_SCREENSHOTS_USE_EXISTING_E2E:-0}" = "1" ]; then
+  echo "Existing live-account screenshot capture is not supported for App Store screenshots because it can expose private rooms or profiles. Use the local fixture instead." >&2
+  exit 64
+fi
+
+export E2E_MATRIX_PORT="${E2E_MATRIX_PORT:-28008}"
+export E2E_HOMESERVER="${E2E_HOMESERVER:-http://127.0.0.1:${E2E_MATRIX_PORT}}"
+
+case "${E2E_HOMESERVER}" in
+  http://127.0.0.1:* | http://localhost:* | http://[::1]:*) ;;
```

Shellcheck (0.11.0)

[info] Ranges can only match single chars (mentioned due to duplicates).

(SC2102)